packit_service/worker/handlers/__init__.py

```diff
@@ -16,7 +16,7 @@
     FedmsgHandler,
     DistGitCommitHandler,
 )
-from packit_service.worker.handlers.github_handlers import (
+from packit_service.worker.handlers.forges_handlers import (
```

Comment:
:tada: Thanks!

Reply:
I am only thinking about the new name -- by now, handlers are forge-independent and can be grouped by the focus. E.g. I like that you have moved the `TestingFarmHandler` to other TF handlers.

(`forges_handlers` now does not make much sense)

Reply:
Ok, let me know if you figure out something better than `forges_handlers` :-)

Reply:
(can be done later/elsewhere)

What about splitting the handlers by purpose -- `copr_build_handlers`/`koji_build_handlers`/`testing_farm_handlers`/...?

(We just need to be sure that the file with handler is always imported to trigger the decorators so we don't ignore them.)

Reply:
Thanks!